spec/sidekiq/evss/disability_compensation_form/submit_form0781_spec.rb

```diff
@@ -103,7 +105,7 @@
         Form526Submission.create(user_uuid: user.uuid,
                                  user_account:,
                                  auth_headers_json: auth_headers.to_json,
-                                 saved_claim_id: saved_claim.id,
+                                 saved_claim_id: saved_claim_with_0781v2.id,
```

Comment:
Are there other instances in the spec, besides this one, where `submission` or other assignments that reference a `saved_claim`, could be updated, like L712?